APIs/schemas/substream_constraints.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Describes substream constraints",
+  "title": "Constraint Set",
+  "type": "object",
+  "minProperties": 1,
+  "properties": {
+    "description": {
+      "description": "Substream group description",
+      "type": "string"
+    },
+    "format": {
+      "description": "Substream format",
+      "type": "string",
+      "anyOf": [
+        {
+          "enum": [
+            "urn:x-nmos:format:video",
+            "urn:x-nmos:format:audio",
+            "urn:x-nmos:format:data"
+          ]
+        },
+        {
+          "pattern": "^urn:x-nmos:format:(?!mux$)"
+        }
+      ]
+    },
+    "count": {
+      "$ref": "param_constraint_number.json"
+    },
```

We've discussed that most/all mux Receivers are capable of discarding substreams they are not interested in. Therefore we started discussing whether a `minimum` attribute could be simpler and sufficient?

E.g. something like

```suggestion
    "minimum": {
      "type": "integer",
      "default": 1,
      "minimum": 0
    },
```